instrumentation/github.com/aws/aws-sdk-go/service/otels3/attributes.go

```diff
@@ -0,0 +1,58 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otels3
+
+import "go.opentelemetry.io/otel/label"
+
+const (
+	operationPutObject    = "PutObject"
+	operationGetObject    = "GetObject"
+	operationDeleteObject = "DeleteObject"
+
+	storageOperationKey   = label.Key("storage.operation")
+	storageDestinationKey = label.Key("storage.destination")
+	storageSystemKey      = label.Key("storage.system")
```

Comment:
Should these be namespaces with `aws`? It seems necessary to avoid conflict.

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/common/attribute-and-label-naming.md#attribute-and-label-naming